pkgs/os-specific/bsd/freebsd/pkgs/libsysinfo.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  mkDerivation,
+  fetchurl,
+  fetchFromGitHub,
+}:
+let
+  pcFile = fetchurl {
+    url = "https://raw.githubusercontent.com/freebsd/freebsd-ports/a613e66a54d54251878412b74c1e99defdac4192/devel/libsysinfo/files/libsysinfo.pc.in";
```

Comment:
has the pc file been removed or broken after this commit?

Reply:
The pc file was never included in upstream. freebsd-ports just has their own copy of it.

Reply:
Can we send a PR upstream (to the GitHub repo)? Seems like something that ought to happen.

Reply:
I think upstream  is effectively abandoned - it was someone's GSoC project a decade ago and then someone imported it to github and it hasn't been touched since. I'll open an issue to see if they feel like pulling in this file...

Reply:
https://github.com/bsdimp/libsysinfo/issues/1